.environment/yarn/run-yarnCheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function note() {
+    echo "yarn-check> info: ${*}"
+}
+
+function yarn_lock_check() {
+    note "Checking Yarn lock integrity"
+    yarn check --integrity
+}
+
+cd frontend-react
+yarn_lock_check
+RC=$?
+
+if [[ ${RC?} != 0 ]]; then
+    echo "yarn-check> ERROR: Your yarn lock file is out of sync."
```

Comment:
Do we want to add "Please run yarn install and try again"?
I like to provide a message where the user doesn't need to think :)